mig/shared/configuration.py

```diff
@@ -670,6 +671,7 @@ class Configuration:
     auto_add_oid_user = False
     auto_add_oidc_user = False
     auto_add_resource = False
+    auto_add_user_permit = [('distinguished_name', '.*')]
```

Comment:
So if I read correctly this is what we would get after parsing the string version of a passed in name. Idea: if the small chunk of code that does the parsing was defined as its own little function, say `def _parse_auto_add_user_permit(): ...` somewhere then we could assign this as calling that function and passing the _same_ default string value as we do for the generateconfs argument.